packages/utils/src/fs/is-file.ts

```diff
@@ -1,11 +1,11 @@
-import { statSync } from 'fs';
+import { accessSync } from 'fs';
 
 /** Check if a path is a file and exists. */
 export const isFile = (filePath?: string): boolean => {
     try {
-        const stats = statSync(filePath!);
+        accessSync(filePath!);
```

Comment:
This is not equivalent to `statSync`. This method will work for a file or a directory, and `isFile` should check if the path is a file.

Reply:
Are you sure? 

[The docs say](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_stat_path_options_callback):

> To check if a file exists without manipulating it afterwards, fs.access() is recommended.

I know that some tests are failing probably because of this but haven't dig into those yet.

Reply:
I've reverted the changes just in case this was the issue with the babel parser.

Also from the docs:

> Using fs.stat() to check for the existence of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended. Instead, user code should open/read/write the file directly and handle the error raised if the file is not available.

Not sure why it's not recommended though 🤔

Reply:
> To check if a file exists without manipulating it 

The thing is that isFile is not checking if a file exists, isFile is checking if the path is a file.

Reply:
> Not sure why it's not recommended though

I don't know neither. I don't like exceptions hehehe

Reply:
Well, I've reverted the code so tests are passing again.